Крестики-нолики_1/PlayField.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include <vector>
+#include <cassert>
+
+using namespace std;
+
+class PlayField
+{
+public:
+	static constexpr int fieldSize = 3;
+	enum Cells
+	{
+		csEmpty,
+		csCross,
+		csNought
+	};
+	enum Status
+	{
+		fsInvalid,
+		fsNormal,
+		fsCrossesWin,
+		fsNoughtsWin,
+		fsDraw
+	};
+
+	class CellPos
+	{
+	public:
+		static CellPos GetCellPos(int pos)
+		{
+			assert(pos >= 0 || pos < fieldSize * fieldSize);
+			return CellPos(pos / fieldSize, pos % fieldSize);
+		}
+
+		CellPos(int i, int j)
+		{
+			assert(i >= 0 || j >= 0 || i <= fieldSize - 1 || j <= fieldSize - 1);
+			x = i;
+			y = j;
+		}
+
+		int GetPos() { return x * fieldSize + y; }
+		int GetX() { return x; }
+		int GetY() { return y; }
+
+	private:
+		int x;
+		int y;
+	};
+
+	Cells operator[] (CellPos) const;
+	vector <CellPos> getEmptyCells() const;

```

Comment:
```suggestion
	vector<CellPos> getEmptyCells() const;

```
После vector пробел не ставят, не принято. Это одна конструкция.